src/main/java/com/iopipe/IOpipeConfiguration.java

```diff
@@ -29,6 +30,55 @@
 	private static final Logger _LOGGER =
 		LogManager.getLogger(IOpipeConfiguration.class);
 	
+	/** Used to compare plugin names. */
+	private static final Comparator<String> _PLUGIN_COMPARATOR =
+		new Comparator<String>()
+		{
+			/**
+			 * {@inheritDoc}
+			 * @since 2018/05/03
+			 */
+			@Override
+			public int compare(String __a, String __b)
+			{
+				// null A is before B
+				if ((__a == null) != (__b == null))
+					return (__a == null ? -1 : 1);
+				
+				// Both are null
+				else if (__a == null)
+					return 0;
+				
+				// Differing length
+				int na = __a.length(),
```

Comment:
can these variables get names more meaningful than `na` and `rv`

Reply:
I changed the `na` and `nb` to `lena` and `lenb`. The variable `rv` is short for return value, I could change it to `retval` or `diff` since it stores the difference (even though the difference is not actually considered, just how the return value compares with zero).

Reply:
I've seen `rv` in the rest of the codebase, so I'm okay with that. All the short letters together were getting confusing :)